Makefile

```diff
@@ -2,19 +2,22 @@
 bin/cobra:
 	GOBIN=$(PWD)/bin go install -mod=readonly $(shell go list -m -f '{{ .Path}}/cobra@{{ .Version }}' github.com/spf13/cobra)
 
-cadctl/cadctl: cadctl/**/*.go pkg/**/*.go go.mod go.sum
-	GOBIN=$(PWD)/cadctl go install -ldflags="-s -w -extldflags=-zrelro -extldflags=-znow" -buildmode=pie -mod=readonly -trimpath $(PWD)/cadctl
+bin/embedmd:
+	GOBIN=$(PWD)/bin go install -mod=readonly github.com/campoy/embedmd@latest
```

Comment:
Can you pin this to a version? If embedmd ever breaks we are wasting time investigating which version did work. 
Just pin it at the version you are currently using